### DISCUSSION TITLE
SAE behindert die Steuerung von Verbrauchern über dem HM2

### DISCUSSION THREAD

**Discussion by sgr4400** [camueller/SmartApplianceEnabler#291] [Installation und Betrieb]
Ich habe die Version 1.6 und darin zwei Verbraucher mit S0-Zählern angelegt - Wärmepumpe und ElektroAuto. Beide sind aktuelle nicht wirklich steuerbar, ich wollte nur gerne den aktuellen Verbrauch im SunnyPortal sehen.

Leider bewirken aber diese beiden neuen Verbraucher, dass ein Heizstab, der mit dem SHM 2 über eine Fritz!Dect Steckdose gesteuert wird, nicht mehr Anspring wenn Überschuss vorhanden ist. 

Warum ist das so?

**Comment by camueller** [comment]
Da musst Du Dich an SMA wenden, wenn der HM2 nicht funktioniert, wenn andere Geräte über SEMP (da ist das SMA-Protokoll, über das HM und SAE kommunizieren) eingebunden sind.
